src/content/docs/en/guides/cms/ghost.mdx

```diff
@@ -286,5 +286,6 @@ To deploy your site visit our [deployment guide](/en/guides/deploy/) and follow
 
 - You can get started with this [Astro Ghost Starter](https://github.com/PhilDL/astro-starter-ghost).
 - Andre Carrera wrote [a blog post](https://andr.ec/posts/astro-ghost-blog/) showing how to use Ghost's SDK to embed posts and previews in your Astro project.
+- Adam Matthiesen wrote [a blog post](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) showing how to deal with SSR changes with Ghost, as well as better layout of the Ghost CONTENT_API.
```

Comment:
```suggestion
- [Astro + Ghost + Tailwind CSS](https://andr.ec/posts/astro-ghost-blog/) by Andre Carrera
- [Ghost CMS & Astro](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) by Adam Matthiesen
```
I updated these to match our more current format, which uses the blog title as better linked text.